packages/react-components/react-infolabel/src/components/InfoLabel/useInfoLabel.ts

```diff
@@ -70,6 +70,7 @@ export const useInfoLabel_unstable = (props: InfoLabelProps, ref: React.Ref<HTML
     infoButton.info = slot.optional(infoButton?.info, {
       defaultProps: {
         id: baseId + '__info',
+        'aria-labelledby': baseId + '__info',
```

Comment:
_suggestion:_ Should you move this into a `infoSlotId` const above that is reused so that if this changes, there is only one place where you have to make the change?

Reply:
heh, I debated this too and ended up not doing it b/c the two strings are immediately adjacent, and none of the other `baseId + '__extraString'` ids were consts. I'm ambivalent though, and happy to change if you think it's worth it!

Reply:
actually, I'll update it. Also added a comment about it for future reference :D